Documentation/Reference/ViewDescription.md

```diff
@@ -55,15 +57,21 @@ let description = UIView.describe { config in
 ### Specifying a subview that should contain any view-backed child elements
 
 ```swift
-let description = MyCustomView.describe { config in
-    config.contentView = { $0.childContainerView }
+private class MyCustomView {
```

Comment:
Does this need to be a `UIView` subclass?

```suggestion
private class MyCustomView: UIView {
```

Reply:
3780d80